src/main/java/com/google/cloud/genomics/utils/Paginator.java

```diff
@@ -203,9 +204,9 @@ private Datasets(Genomics genomics) {
       super(genomics);
     }
 
-    @Override Genomics.Datasets.List createSearch(Genomics.Datasets api, Long request,
+    @Override Genomics.Datasets.List createSearch(Genomics.Datasets api, String request,
```

Comment:
@melissachang I renamed it.  createSearch exists many times in that file though.  Can you PTAL and let me know if I understood you correctly?